daviaugustos/chat-app
# TL-03: The user should be able to write a text message and send it to the recipient.

# Title

[TL-03](https://trashlab.notion.site/Mobile-Take-Home-Assignment-10bdb96ac1e78033af18cdfc06a04fee): The user should be able to write a text message and send it to the recipient.

## Description

💬 Implemented user interface for sending and receiving messages logic.

## Screenshots

![image](https://github.com/user-attachments/assets/6fc29f5f-684b-4c51-b24b-4d9bcd512f8b)
![image](https://github.com/user-attachments/assets/a1ced6eb-d7ea-4d07-9f22-577dac506871)

## Testing

- Open the app;
- Click on a contact to open the respective chat message history.
- Type a message and hit the save button;
- Check if the message sent is added to the chat history.
